src/main/java/org/sagebionetworks/web/client/widget/entity/file/FileDownloadMenuItemView.java

```diff
@@ -10,9 +10,9 @@
 
 	void setIsDirectDownloadLink(String href);
 
-	void setIsUnauthenticatedS3DirectDownload();
+	void showDownloadError();
```

Comment:
Rename (I'd expect this to show a download error by calling).
Perhaps to something like
setIsSFTPDownload() (to be aligned with the other method names in this view)